### PR TITLE
feat(body-limit): Replace c.req.raw with body limit middleware proxy

### PR DESCRIPTION
**Pull request by usualoma**
### Author should do the followings, if applicable

- [x] Add tests
- [x] Run tests
- [x] `yarn denoify` to generate files for Deno

### CODE REVIEW

**Comment by rafaell-lycan** [comment]
That's a very clever move.

Btw, Express [supports up to 100kb](https://expressjs.com/en/resources/middleware/body-parser.html#limit) by default, and it's quite easy to configure on its middleware.

**Comment by usualoma** [comment]
@yusukebe ready for review!

File: deno_dist/middleware/body-limit/index.ts [hunk 1]
```diff
@@ -25,42 +33,57 @@ type BodyLimitOptions = {
  * )
  * ```
  */
-export const bodyLimit = (options: BodyLimitOptions): MiddlewareHandler =>
-  async function bodyLimit(c, next) {
-    if (!c.req.raw.body) return next()
+export const bodyLimit = (options: BodyLimitOptions): MiddlewareHandler => {
+  const onError: OnError = options.onError || ((c) => c.text('413 Request Entity Too Large', 413))
```
Comment (yusukebe) [inline deno_dist/middleware/body-limit/index.ts:37]: It is better to make `'413 Request Entity Too Large'` as a constant to reuse in line 66.

**Comment by yusukebe** [comment]
@usualoma Thanks!

I've left a comment.

File: deno_dist/middleware/body-limit/index.ts (hunk 1 above)
Comment (usualoma) [reply deno_dist/middleware/body-limit/index.ts]: @yusukebe Thank you!

refactored in 2a326ca

The message has been changed to "Payload Too Large" in accordance with the RFC.
https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.11

Comment (yusukebe) [reply deno_dist/middleware/body-limit/index.ts]: Great 👍

**Review by yusukebe — APPROVED** [review]
LGTM! Merge now.

**Pull request merged**